substrate/frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -427,8 +429,10 @@ pub trait MinerConfig {
 	///
 	/// The weight is computed using `solution_weight`.
 	type MaxWeight: Get<Weight>;
-	/// The maximum number of winners that can be elected.
-	type MaxWinners: Get<u32>;
+	/// The maximum number of winners that can be elected per page (and  overall).
+	type MaxWinnersPerPage: Get<u32>;
```

Comment:
Similarly, note that this miner code is only capable of generating a solution with 1 page.